### DISCUSSION TITLE
How to update only a part of the fields?

### DISCUSSION THREAD

**Discussion by secsilm** [milvus-io/milvus#32835] [Q&A and General discussion] [answered]
For example, for the existing fields `id`, `A`, `B`, `C`, I only want to update the field `A` where id is 0:

```python
client.update(collection_name, data=[{'id': 0, 'A': 'new_value'}])
```

**Comment by yhmo** [comment, marked as answer]
Currently, Milvus only supports updating the entire row. Partial update is not supported, we have engineers working in this feature, maybe available in v2.5 or v3.0.

**Reply by xiaofan-luan** [reply]
there is some discussion about this feature but yes current you have to do it by read then write
